boards/arm/frdm_k82f/Kconfig.defconfig

```diff
@@ -20,16 +20,6 @@ config MCG_VDIV0
 config MCG_FCRDIV
 	default 1
 
-if FLASH
```

Comment:
I'll leave the `select` vs `depends on` discussion to others, just wanted to make a note that the K82F changes here are valid given the other changes in this PR.